packages/design/src/DataTable/Cells.tsx

```diff
@@ -2,26 +2,60 @@ import React from 'react';
 import { Label } from 'design';
 import * as Icons from 'design/Icon';
 import { displayDate } from 'shared/services/loc';
-import { SortDir } from './types';
+import { ServersideConfig, SortDir, TableColumn } from './types';
 
 export const Cell = props => <td children={props.children} {...props} />;
 
-export const SortHeaderCell = ({ dir, text, onClick }: SortHeaderCellProps) => {
+export function SortHeaderCell<T>({
+  column,
+  serverside,
```

Comment:
`serverside` is a confusing name here. Can we think of a better name?

Reply:
hm, how about `serversideProps`?

Reply:
Better, but still a bit confusing. These props are used for filtering and sorting right? Can we somehow include that in the name?